File: Models/ProductItem.cs
namespace WebShopInc.Models
{
    public class ProductItem
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }

        public string ImgURL { get; set; }

        public string Unit { get; set; }

        public List<ProductDeliveryTime> deliveryTimeList { get; set; }

    }
}

File: ClientApp/src/app/ProductItem.ts

export interface ProductItem {
  id: number,
  name: string,
  description: string,
  imageUrl: string,
  unit: string,
  productDeliveryTimes: ProductDeliveryTime[]
}

export interface ProductDeliveryTime {
  fromDays: number,
  toDays: number,
  days: number,
}
